auto-bazaar-pricer.user.js

```diff
@@ -169,12 +169,20 @@ $(() => {
              * Create element and add to page.
              */
             build: function () {
+                var $container = $('[class^="linksContainer_"');
+                var $link = $('[class^="linkContainer_"');
+                var classes = '';
+                for (const cls of $link.get(0).classList.values()) {
+                    if (cls.indexOf('__') >= 0) {
+                        classes += cls + ' ';
+                    }
+                }
```

Comment:
```suggestion
                const container = $('[class^="linksContainer_"'),
                    link = $('[class^="linkContainer_"'),
                    classes = link[0].className;
```
Rather than looping over all the separate classes and concatenating a string, you could just get the full string by using .className instead.

Reply:
nice optimalisation you're very right! looping over the classes is absolutely unnecessary here